addons/mail/static/tests/thread/unread_messages_banner.test.js

```diff
@@ -112,46 +109,6 @@ test("reset new message separator from unread messages banner", async () => {
     await contains("span", { text: "30 new messagesMark as Read", count: 0 });
 });
 
-test("scroll to unread notification", async () => {
```

Comment:
Shouldn't it still be possible to scroll to a notification but through another flow?

Reply:
I don't see one as notifications can't be unread nor pinned.

Reply:
Even if it's not in the UI, couldn't a user still create a pinned message from it through RPC? Are we preventing it?

I guess if everything else fails, one could still craft the message link with the correct id.

I don't know, it just feels off that we are arbitrarily limiting all interactions with notifications.

Reply:
We're not limiting it, notification still register their ref to their parent thread so it would still work technically but there is no way to test it in the UI. I guess I could craft it manually to keep the test if you want to.

Reply:
What's the easiest way? Just to have it pinned in data? That seems like a small enough amount of "crating" to keep the test.